Materialy/TestowanieBazDanych/TestowanieBazDanychNarzedzia.md

```diff
@@ -0,0 +1,14 @@
+[![](https://img.shields.io/badge/Facebook-%23TestowanieOprogramowania-blue.svg)](https://www.facebook.com/groups/TestowanieOprogramowania/)
+
+
+# Testowanie baz danych
+
+
+## Narzędzia - Open Source
+
+* DbFit http://dbfit.github.io/dbfit/index.html
+   * Test-driven database development. Write readable, easy-to-maintain unit and integration tests for your database code.
```

Comment:
Narzędzie wygląda na przeznaczone zdecydowanie bardziej inżynierom tworzącym bazy danych. Jaki zysk będą mieli z niego dokładniej testerzy?